scripts/github/generate_release_text.rb

```diff
@@ -19,12 +19,13 @@
   trim_mode: '<>'
 )
 
-# get last polkadot version. Use handy Gem::Version for sorting by version
+# get last full polkadot version. Use handy Gem::Version for sorting by version
+# Note: This means ^v1.2.3$, explicitly ignoring -rc versions
 last_version = pg
               .tags
               .map(&:name)
-              .grep(/^v\d+\.\d+\.\d+.*$/)
-              .sort_by { |v| Gem::Version.new(v.slice(1...)) }[-2]
+              .grep(/^v\d+\.\d+\.\d+$/)
+              .max_by { |v| Gem::Version.new(v.slice(1...)) }
```

Comment:
TODO: Replace this with a snippet to detect the last published release rather than relying on git tags